src/Components/Buttons/Facebook.tsx

```diff
@@ -4,19 +4,23 @@ import Icon from "../Icon"
 import Button, { ButtonProps } from "./Default"
 
 const FacebookButton = (props: ButtonProps) => {
-  const icon = <Icon name="facebook" color="white" />
-  return <Button {...props} icon={icon}>{props.children || 'Log in with Facebook'}</Button>
+  const icon = <Icon name="facebook" color={props.color || "white"} />
+  return (
+    <Button {...props} icon={icon}>
+      {props.children || "Log in with Facebook"}
+    </Button>
+  )
 }
 
 export default styled(FacebookButton)`
-  background: #39439C;
+  background: #39439c;
```

Comment:
Whoah what are these mysterious colors lol. Are they repeated anywhere? We might want to add to that Colors file for default social colors if they are used elsewhere.